prime-router/src/main/kotlin/transport/GAENTransport.kt

```diff
@@ -240,8 +240,11 @@ class GAENTransport : ITransport, Logging {
         } else {
             // The follow error table is based on ENCV server docs.
             val postResult = when (response.statusCode) {
-                400 -> PostResult.FAIL // Bad parameters
-                409 -> PostResult.SUCCESS // UUID already present (consider this a success)
+                // Note: To keep us not to post error in the Last Mile Failure page, we are going to ignore
```

Comment:
Just adding a suggestion here to improve this comment for clarity. Something like "The majority of GAEN errors are invalid date content errors that can not be rectified by resubmission via the Last Mile Failures page. We have decided to ignore the 400 error code for GAEN messages as we are not taking action on these errors. This does not affect ELR messages sent to STLTs."